tests/unit/test_table.py

```diff
@@ -3752,7 +3752,7 @@ def test___repr___explicit(self):
         time_partitioning = self._make_one(
             type_=TimePartitioningType.DAY, field="name", expiration_ms=10000
         )
-        expected = "TimePartitioning(" "expirationMs=10000," "field=name," "type=DAY)"
+        expected = "TimePartitioning(" "expiration_ms=10000," "field=name," "type_='DAY')"
```

Comment:
(nit)
Three implicitly concatenated strings are a leftover from a past blacken session and we can get rid of it while already modifying this line.
```suggestion
        expected = "TimePartitioning(expiration_ms=10000,field=name,type_='DAY')"
```